src/commonMain/kotlin/komem/litmus/LTDefinition.kt

```diff
@@ -30,11 +29,16 @@ class LTDefinitionScope<S>(
     }
```

Comment:
Maybe add checks that `spec` and `outcome` are only called once during test configuration?

Reply:
Will do

On this topic: I wanted to put a DslMarker annotation here, but I couldn't figure out how. Does it require different receiver classes to work?

Reply:
Could you point to your code with DslMarker?
I can have a look.